website/server/controllers/api-v3/quests.js

```diff
@@ -106,11 +106,15 @@ api.inviteToQuest = {
     let inviterVars = getUserInfo(user, ['name', 'email']);
     let membersToEmail = members.filter(member => {
       // send push notifications while filtering members before sending emails
-      sendPushNotification(
-        member,
-        common.i18n.t('questInvitationTitle'),
-        common.i18n.t('questInvitationInfo', { quest: quest.text() })
-      );
+      if (member.preferences.pushNotifications.invitedQuest !== false) {
+        sendPushNotification(
+          member,
+          common.i18n.t('questInvitationTitle'),
+          common.i18n.t('questInvitationInfo', {quest: quest.text()}),
```

Comment:
quest.text should be passed `req.language`. In general where available you should use `res.t()` instead of `i18n.t` so the correct language will be applied